prime-router/src/main/kotlin/cli/LivdTableDownload.kt

```diff
@@ -240,4 +288,19 @@ class LivdTableDownload : CliktCommand(
         TermUi.echo("\tSUCCESS: The $outputfile file is created.\n")
         return true
     }
+
+    /**
+     * Updates the LIVD lookup table name [livdLookupTable] of CSV file.  It setups PRIME CLI Lookup Table Create
+     * Command line options.  And then, it calls the create lookup table command to create the new version of lookup
+     * table.  Note, it always create the new version regardless since it uses -f option.
+     */
+    private fun updateTheLivdLookupTable(livdLookupTable: File) {
+        val tableName = livdLookupTable.nameWithoutExtension
```

Comment:
Let's hardcode the name of this table since it is meant to always be the same.  How about just "LIVD-SARS-CoV-2"?  Define this at the top of the class.

Reply:
Done.